app/router.js

```diff
@@ -11,6 +11,11 @@ Router.map(function() {
     this.route("featured");
     this.route("all");
   });
+
+  this.resource('users', function () {
+    this.resource('user', { path: ':user_id' });
+  });
```

Comment:
We can decide if this needs to be simplified (i.e. not nested) once we have a clearer idea of what we'll get form the api.